### PR TITLE
Fixes #1079 - added functional tests for URL and search box update up…

### LINKED ISSUE #1079: Functional tests for label Search
Things to test:

- [x]  clicking on a label in /issues updates the URL and appends &q=label:<clicked label>
- [x]  clicking on a label in /issues updates the search box

### PR DESCRIPTION
**Pull request by deepthivenkat**
…on label click in /issues

r? @miketaylr 

The first test checks the URL update using .getCurrentUrl.

The second test checks the search box update by getting value of search box using Id selector

### CODE REVIEW

File: tests/functional/search-non-auth.js [hunk 1]
```diff
@@ -66,6 +66,35 @@ define([
         .end();
     },
 
+    'Clicking on label search adds query parameter to the URL': function() {
+      var params = '?q=dfjdkfjdkfjkdfjdkjf';
```
Comment (miketaylr) [inline tests/functional/search-non-auth.js]: I think we can remove `params` here and in the test below (as well as from `require.toUrl(url('/issues') + params)` -- it's not doing anything in these tests.

**Comment by miketaylr** [comment]
r+ with the one small nit addressed.

File: tests/functional/search-non-auth.js (hunk 1 above)
Comment (deepthivenkat) [inline tests/functional/search-non-auth.js]: Done!

**Comment by miketaylr** [comment]
nice.

**Pull request merged**